### DISCUSSION TITLE
帮忙看看这个query profile的瓶颈在哪里？如何优化？

### DISCUSSION THREAD

**Discussion by yuanoOo** [apache/doris#16253] [A - General / Q&A]
Fragment  1:
            Instance  35d902d00bf44e6e-91ccf1151d80c76f  (host=TNetworkAddress(hostname:,  port:9060)):(Active:  49m35s,  %  non-child:  0.16%)
                  -  FragmentCpuTime:  48m43s
                  -  MemoryLimit:  32.00  GB
                  -  PeakMemoryUsage:  17.92  GB
                  -  RowsProduced:  1.412097492B  (1412097492)
                BlockMgr:
                      -  BlockWritesOutstanding:  0
                      -  BlocksCreated:  0
                      -  BlocksRecycled:  0
                      -  BufferedPins:  0
                      -  BytesWritten:  0.00  
                      -  MaxBlockSize:  8.00  MB
                      -  TotalBufferWaitTime:  0ns
                      -  TotalEncryptionTime:  0ns
                      -  TotalIntegrityCheckTime:  0ns
                      -  TotalReadBlockTime:  0ns
                VDataStreamSender  (dst_id=5,  dst_fragments=[{"35d902d00bf44e6e-91ccf1151d80c786"}]):(Active:  2m,  %  non-child:  3.84%)
                      -  BytesSent:  0.00  
                      -  IgnoreRows:  0
                      -  LocalBytesSent:  251.83  GB
                      -  OverallThroughput:  0.0  /sec
                      -  PeakMemoryUsage:  0.00  
                      -  SerializeBatchTime:  0ns
                      -  UncompressedRowBatchSize:  0.00  
                VAGGREGATION_NODE  (id=3):(Active:  47m29s,  %  non-child:  56.81%)
                      -  ExecOption:  Streaming  Preaggregation
                      -  BuildTime:  29m10s
                      -  ExecTime:  2s718ms
                      -  ExprTime:  26m35s
                      -  GetResultsTime:  0ns
                      -  MergeTime:  0ns
                      -  PeakMemoryUsage:  10.00  MB
                      -  RowsReturned:  1.412097492B  (1412097492)
                      -  RowsReturnedRate:  495.514K  /sec
                    VHASH_JOIN_NODE  (id=2):(Active:  17m41s,  %  non-child:  32.78%)
                          -  BuildBuckets:  67.108864M  (67108864)
                          -  PeakMemoryUsage:  17.80  GB
                          -  PushDownComputeTime:  15s424ms
                          -  PushDownTime:  19.988us
                          -  RowsReturned:  1.412107881B  (1412107881)
                          -  RowsReturnedRate:  1.329739M  /sec
                        ProbePhase:
                              -  ProbeExprCallTime:  30.767ms
                              -  ProbeFindNextTime:  384.457ms
                              -  ProbeRows:  99.526531M  (99526531)
                              -  ProbeTime:  16m16s
                              -  ProbeWhenBuildSideOutputTime:  4m27s
                              -  ProbeWhenProbeSideOutputTime:  9m47s
                              -  ProbeWhenSearchHashTableTime:  30s173ms
                        BuildPhase:
                              -  BuildExprCallTime:  32.337us
                              -  BuildRows:  135.566239M  (135566239)
                              -  BuildTableExpanseTime:  0ns
                              -  BuildTableInsertTime:  28s330ms
                              -  BuildTableTime:  28s523ms
                              -  BuildTime:  1m25s
                        VEXCHANGE_NODE  (id=4):(Active:  30s171ms,  %  non-child:  0.96%)
                              -  BytesReceived:  3.00  GB
                              -  DataArrivalWaitTime:  22s834ms
                              -  DeserializeRowBatchTimer:  9s683ms
                              -  FirstBatchArrivalWaitTime:  10.410ms
                              -  PeakMemoryUsage:  17.23  MB
                              -  RowsReturned:  135.566239M  (135566239)
                              -  RowsReturnedRate:  4.493244M  /sec
                              -  SendersBlockedTotalTimer(*):  5m59s
                        VOLAP_SCAN_NODE  (id=0):(Active:  356.177ms,  %  non-child:  0.01%)
                              -  BatchQueueWaitTime:  170.988ms
                              -  BytesRead:  1.54  GB
                              -  NumDiskAccess:  1
                              -  NumScanners:  1
                              -  PeakMemoryUsage:  99.91  MB
                              -  RowsRead:  99.526531M  (99526531)
                              -  RowsReturned:  99.526531M  (99526531)
                              -  RowsReturnedRate:  279.429686M  /sec
                              -  ScannerBatchWaitTime:  1m8s
                              -  ScannerSchedCount  :  6.714K  (6714)
                              -  ScannerWorkerWaitTime:  307.435ms
                              -  TabletCount  :  1
                              -  TotalReadThroughput:  0.0  /sec
                            OlapScanner:
                                  -  VconjunctExprTree:  VectorizedFn[eq({"__DORIS_DELETE_SIGN__",  "Int8"})]{SlotRef(slot_id=60  type=TINYINT  codegen=false)  column  id:  9,  name:  __DORIS_DELETE_SIGN__,  is  nulable:  0,  type=TINYINT  codegen=false}
                                  -  BlockConvertTime:  0ns
                                  -  BlockFetchTime:  1m6s
                                  -  ReaderInitTime:  12.654ms
                                  -  RowsDelFiltered:  0
                                  -  RowsPushedCondFiltered:  0
                                  -  ScanCpuTime:  1m6s
                                  -  ScanTime:  0ns
                                  -  ShowHintsTime_V1:  0ns
                                SegmentIterator:
                                      -  BitmapIndexFilterTimer:  37.530us
                                      -  BlockLoadTime:  1m6s
                                      -  BlockSeekCount:  95
                                      -  BlockSeekTime:  23.597ms
                                      -  BlocksLoad:  24.309K  (24309)
                                      -  CachedPagesNum:  63.886K  (63886)
                                      -  CompressedBytesRead:  0.00  
                                      -  DecompressorTimer:  0ns
                                      -  IOTimer:  0ns
                                      -  IndexLoadTime_V1:  0ns
                                      -  NumSegmentFiltered:  0
                                      -  NumSegmentTotal:  9
                                      -  RawRowsRead:  99.527668M  (99527668)
                                      -  RowsBitmapIndexFiltered:  0
                                      -  RowsBloomFilterFiltered:  0
                                      -  RowsConditionsFiltered:  0
                                      -  RowsKeyRangeFiltered:  0
                                      -  RowsStatsFiltered:  0
                                      -  RowsVectorPredFiltered:  1.137K  (1137)
                                      -  TotalPagesNum:  63.886K  (63886)
                                      -  UncompressedBytesRead:  0.00  
                                      -  VectorPredEvalTime:  0ns
                            RuntimeFilter:in_or_bloomfilter:
                                  -  RealRuntimeFilterType:  bloomfilter
                                  -  HasPushDownToEngine:  true
                                  -  AWaitTimeCost:  0ns
                                  -  EffectTimeCost:  1m25s

**Comment by gnehil** [comment]
Please add my WeChat to analyze the specific situation.
WeChat: gnehil489
